deploy/backup/copy-backup.sh

```diff
@@ -35,10 +35,25 @@ get_backup_dest() {
     local backup=$1
 
     if $ctrl get "pxc-backup/$backup" 1>/dev/null 2>/dev/null; then
+        BASE64_DECODE_CMD=""
+        echo $(echo 'abc'|base64) | base64 -d 1>/dev/null 2>/dev/null
+        res1=$?
+        echo $(echo 'abc'|base64) | base64 -D 1>/dev/null 2>/dev/null
+        res2=$?
+
+        if [[ ${res1} == 0 ]]; then
+            BASE64_DECODE_CMD="base64 -d"
+        elif [[ ${res2} == 0 ]]; then
```

Comment:
@delgod You're right, so I added the command detection code before executing the command, and I passed tests on common systems, pls review the code again. #142